crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -207,8 +208,28 @@ impl Room {
     }
 
     /// Is this room considered a direct message.
-    pub fn is_direct(&self) -> bool {
-        !self.inner.read().unwrap().base_info.dm_targets.is_empty()
+    pub async fn is_direct(&self) -> StoreResult<bool> {
```

Comment:
```suggestion
    #[instrument(skip_all, fields(room_id = ?self.room_id))]
    pub async fn is_direct(&self) -> StoreResult<bool> {
```

Reply:
Just for my curiosity: what this attribute is supposed to do?